### PR TITLE
Nick/mobile attestations

### PR DESCRIPTION
**Pull request by nick**
Start of removing cookie dependency and allowing redirection for twitter / facebook so it works on mobile.

### CODE REVIEW

**Comment by franckc** [comment]
@shahthepro and @nick: This is shaping up nicely. Good work !

Question for you: This coming Wed 5/1, we were hoping to promote to our users the May campaign of the Origin Rewards program (which includes a 25 OGN reward per attestation) and to encourage them to use the Origin mobile app. Do you think this PR can get merged in and tested by Wed 5/1 ? Thanks.

**Comment by shahthepro** [comment]
@franckc I have tested them and they work just fine. @nick is also testing them out, I guess. Once he is done, it can be merged and deployed to staging.

**Comment by franckc** [comment]
@shahthepro You rock. Thank you sir.

File: packages/graphql/src/mutations/attestations/verifyTwitter.js [hunk 1]
```diff
@@ -9,14 +9,15 @@ async function verifyTwitter(_, { identity, authUrl, redirect, code }) {
...
-  const getAuthUrl = `${bridgeServer}/api/attestations/twitter/auth-url?redirect=http%3A%2F%2F172.22.95.23%3A3000%2F%23%2Fprofile%2Ftwitter`
-  const response = await fetch(getAuthUrl, {
-    headers: { 'content-type': 'application/json' }
-  })
-  const authData = await response.json()
-  authUrl = authData.url
-  // }
+  if (!authUrl) {
+    const getAuthUrl = `${bridgeServer}/api/attestations/twitter/auth-url`
+    console.log(redirect)
```
Comment (franckc) [inline packages/graphql/src/mutations/attestations/verifyTwitter.js]: nit: remove this debug statement

**Pull request merged**